components/Browse.tsx

```diff
@@ -551,6 +553,7 @@ export const Browse = () => {
       })
       console.log(e)
     } finally {
+      setClaimingRental(false)
```

Comment:
just pointing out that technically you can still game this because this transaction could finish and right before you could click claim on the next one before they reload

Reply:
also was just thinking can you just use the claim links to get around the single limit also